docs/rules/operator-linebreak.md

```diff
@@ -0,0 +1,121 @@
+# Operator Linebreak (operator-linebreak)
+
+When a statement is too long to fit on a single line, line breaks are generally inserted next to the operators separating expressions. The first style coming to mind would be to place the operator at the end of the line, following the english punctuation rules.
+
+```js
+var fullHeight = borderTop +
+                 innerHeight +
+                 borderBottom;
+```
+
+Some developers find that placing operators at the beginning of the line makes the code more readable.
+
+```js
+var fullHeight = borderTop
+               + innerHeight
+               + borderBottom;
+```
+
+## Rule Details
+
+The `operator-linebreak` rule is aimed at enforcing a particular operator line break style. As such, it warns whenever it sees a binary operator or assignment that does not adhere to a particular style: either placing operators after or before the lines.
+
+### Options
+
+The rule takes an option, a string, which could be either "after" or "before". The default is "after".
+
+You can set the style in configuration like this:
+
+```json
+"operator-linebreak": [2, "before"]
+```
+
+#### "after"
+
+This is the default setting for this rule. This option requires that the operator be placed after and be in the same line as the last expression of the line.
```

Comment:
I'm a bit confused by this description. Shouldn't this just say that the line break should be placed after the operator? Not sure what "last expression of the line" means.